cfp.html

```diff
@@ -128,6 +128,10 @@ <h2>Topics of Interest</h2>
                                 <li>Visualization of or to support software development activities (e.g., software testing)</li>
                                 <li>Visualization-based techniques in software engineering education</li>
                             </ul>
+                            <p>
+                                Papers presenting studies, tools, datasets, or innovative ideas are welcome.
+                                Experimental evaluation of new tools and ideas is desirable, but it is not a precondition to submit a paper to the workshop.
```

Comment:
The last sentence isn't related to the submission/evaluation instead of topics of interest?

Reply:
I'm not sure. To me, this is more about "scope", and the place we are talking about scope is the topics of interest.

Reply:
Ah, ok, this is not the the list of topics... sorry. that's fine then. =D

Reply:
Felipe, it is in the topics. Not in the list, but in the same section.

Reply:
hm, the topics shouldn´t be under the <ul>
but I still think it´s weird to be listed as a topic...

Reply:
I don't understand your comment.

Reply:
Please ignore my comments. I was misunderstanding it. It's all right the way it is.

Reply:
Ok!